### PR TITLE
refactor(dockerfile): update to 0.8.0 and simplify dockerfile

### PR DESCRIPTION
**Pull request by qoomon**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+FROM openjdk:8-jre-alpine
 
-ENV CEREBRO_VERSION 0.7.3
-RUN cd /opt/ \
-    && wget -O cerebro-${CEREBRO_VERSION}.tgz https://github.com/lmenezes/cerebro/releases/download/v${CEREBRO_VERSION}/cerebro-${CEREBRO_VERSION}.tgz \
-    && tar zxvf cerebro-${CEREBRO_VERSION}.tgz \
-    && rm cerebro-${CEREBRO_VERSION}.tgz \
-    && mkdir cerebro-${CEREBRO_VERSION}/logs \
-    && mv cerebro-${CEREBRO_VERSION} cerebro
+RUN apk add bash
```
Comment (moliware) [inline Dockerfile]: This doesn't work for me (Mac) if I don't add:
```
RUN apk update
```
before this line

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
+ENV CEREBRO_VERSION 0.8.0
+ADD ./cerebro-${CEREBRO_VERSION}.tgz /opt
```
Comment (moliware) [inline Dockerfile]: Does this mean that we need a local file named cerebro-${CEREBRO_VERSION}.tgz before building the image? If so, won't it break the automatic process of docker hub?

**Comment by qoomon** [comment]
sorry committed the wrong version, one moment

**Comment by moliware** [comment]
it still doesn't work for me. I had to:

- Create /opt before downloading tgz file (`RUN mkdir /opt` )
- Untar file (`RUN tar zxvf /opt/cerebro-${CEREBRO_VERSION}.tgz -C /opt`) before moving it

After those changes the image is built and weight ~200MB less :)

would you do those changes? or should I add them? 
thank you!

**Comment by qoomon** [comment]
I will do it this evening.

**Comment by qoomon** [comment]
Already done  :-D

**Comment by moliware** [comment]
Thank you!

**Comment by nqv** [comment]
I think this PR make it worse
```
$ docker version
Client:
 Version:      17.06.0-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:31:53 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.06.0-ce
 API version:  1.30 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:51:55 2017
 OS/Arch:      linux/amd64
 Experimental: true

$ docker build .
...
tar: can't open '/opt/cerebro-0.8.1.tgz': No such file or directory
The command '/bin/sh -c tar zxvf /opt/cerebro-${CEREBRO_VERSION}.tgz -C /opt && mv /opt/cerebro-${CEREBRO_VERSION} /opt/cerebro' returned a non-zero code: 1
```
Why keeping the archive file in docker image? Have you ever checked the size of docker images by using `RUN mv` separately?

**Comment by nqv** [comment]
I try to fix it in #304

**Pull request merged**